lib/fun_with_flags/store/serializer/redis.ex

```diff
@@ -5,7 +5,7 @@ defmodule FunWithFlags.Store.Serializer.Redis do
 
   @type redis_hash_pair :: [String.t]
 
-  @spec serialize(FunWithFlags::Gate.t) :: redis_hash_pair
+  @spec serialize(FunWithFlags.Gate.t()) :: redis_hash_pair
```

Comment:
Hi, sorry, can you remove the parenthesis from this, so that it's just `FunWithFlags.Gate.t`. That's the style I've used in the all the typespecs in this codebase.